include/GetInitData.hh

```diff
@@ -0,0 +1,17 @@
+#include <iostream>
+
+#include "TFile.h"
+#include "TString.h"
+#include "TTree.h"
+#include <string>
+
+using namespace std;
+
+#include "G4SystemOfUnits.hh"
+#include "AAMCC.hh"
+
+#include "URun.h"
+#include "UEvent.h"
+#include "UParticle.h"
+#include "BasicTypes.h"
+#include "EventInitialState.h"
```

Comment:
Why did you made a class without fields, if it will be callable object it can be function instead